src/DynamoCoreWpf/Views/PackageManager/PackageManagerSearchView.xaml

```diff
@@ -930,20 +1090,20 @@
                                     FontFamily="{StaticResource ArtifaktElementRegular}"
                                     FontSize="10"
                                     Foreground="#F5F5F5"
-                                    Text="{Binding Path=DownloadState, Converter={StaticResource PackageDownloadStateToStringConverter}}" />
+                                    Text="Upload Error" />
 
                                 <!--  Version  -->
                                 <TextBlock
-                                    Name="version"
+                                    Name="iversion"
```

Comment:
What does `iversion` mean?

Reply:
`infected version` just a name for the text block, no relation to apple 😛